noetic/distribution.yaml

```diff
@@ -2810,6 +2810,13 @@ repositories:
       url: https://github.com/ros-perception/image_transport_plugins.git
       version: noetic-devel
     status: unmaintained
+  imu_from_ios_sensorlog:
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/pietrocolombo/imu_from_ios_sensorlog-release.git
+      version: 0.0.1-1
+    status: unmaintained
```

Comment:
If this is unmaintained, we probably shouldn't release it into the distro.

Reply:
@pietrocolombo Friendly ping. If this is not actually unmaintained, then this status should be changed to `developed` (if it is actively being worked on) or `maintained`. Otherwise, I agree that we probably shouldn't release this package.

I noticed that the package does not declare a license. Please update the `package.xml` to declare a license and add a LICENSE file (see [CONTRIBUTING](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-package-contents) for more details).

Reply:
Thanks, I update the status.

Reply:
Thanks, I have also updated the license.